WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt

```diff
@@ -313,14 +313,15 @@ sealed class MySiteCardAndItem(open val type: Type, open val activeQuickStartIte
                         val numberOfAnswers: Int,
                         val isAnswered: Boolean,
                         val promptId: Int,
+                        val tagUrl: String,
                         val attribution: BloggingPromptAttribution,
                         val showViewMoreAction: Boolean,
                         val showRemoveAction: Boolean,
                         val onShareClick: (String) -> Unit,
                         val onAnswerClick: (PromptID) -> Unit,
                         val onSkipClick: () -> Unit,
                         val onViewMoreClick: () -> Unit,
-                        val onViewAnswersClick: ((PromptID) -> Unit)?,
+                        val onViewAnswersClick: ((String) -> Unit)?,
```

Comment:
💡 Would it make sense to have a more explicit type to make it clear that this `String` represents a tag URL?

e.g.
```
typealias TagURL = String
```